pragma-node/src/handlers/stream/common.rs

```diff
@@ -38,7 +38,7 @@ pub async fn get_historical_entries(
         )
         .await
         .map_err(EntryError::from)?,
-        AggregationMode::Mean | AggregationMode::Twap => unreachable!(),
+        AggregationMode::Twap => unreachable!(),
```

Comment:
I forgot - why is this unreachable? I don't think how it really is unreachable with the current code

Reply:
cause we return an error early if the agg mode is twap

Reply:
where? cant see it

Reply:
Aaaah it's just an util function not an actual endpoint - confused because it was in `handlers` and not in `repository`